[WIP] `Changelog` - Content Design (#1706)
hashicorp/design-system: website/docs/whats-new/release-notes/partials/design-tokens.md

 **`npm package: @hashicorp/design-system-tokens`**

## 1.8.0

**Month Day, Year**

- `Design tokens` added color tokens for “Vault Secrets” product. 

- `IconTile` updated component to include `vault-secrets` product option.

[(#1640)](https://github.com/hashicorp/design-system/pull/1640) Thanks [@didoo](https://github.com/didoo)!

## 1.7.0

**Month Day, Year**

> - `marketing` added JSON output format for target (and in the process refactored internal logic for tokens generation). [(#1513)](https://github.com/hashicorp/design-system/pull/1513) Thanks [@dstaley](https://github.com/dstaley)! 

Reviewer: Token entries are different than component entries, where there may not be a good element to list at the beginning. We should aim for clarity vs trying to align too much to one specific format. I'll add a note about this in the instructions doc.

```suggestion
- Added JSON output format for `marketing` target and refactored internal logic for token generation. [#1513](https://github.com/hashicorp/design-system/pull/1513) Thanks [@dstaley](https://github.com/dstaley)!
```

## 1.6.0

**Month Day, Year**

> - Add design token for loading state icon on search input. [(#1452)](https://github.com/hashicorp/design-system/pull/1452) Thanks [@alex-ju](https://github.com/alex-ju)! 

Reviewer: We should have consistency in the tense used across the entries. I'll add a note to the instructions doc about this. 

```suggestion
- Added design token for loading state icon on search input. [#1452](https://github.com/hashicorp/design-system/pull/1452) Thanks [@alex-ju](https://github.com/alex-ju)! 
```

## 1.5.0

**Month Day, Year**

> - `TooltipButton` added new component and `hds-tooltip` modifier.
> 
> - `Tooltip` added design tokens.
> 
> - [(#1283)](https://github.com/hashicorp/design-system/pull/1283) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!

Reviewer: Another instance where a component change and token change happened in the same PR. I've removed the component change and cleaned up the formatting to match the other entries (PR link and contribution acknowledgment inline).

```suggestion
- Added design tokens for `Tooltip`. [#1283](https://github.com/hashicorp/design-system/pull/1283) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!
```

## 1.4.2

**Month Day, Year**

> - `SideNav` added design tokens [(#1313)](https://github.com/hashicorp/design-system/pull/1313) Thanks [@didoo](https://github.com/didoo)! 

Reviewer: ```suggestion
- Added design tokens for `SideNav` [#1313](https://github.com/hashicorp/design-system/pull/1313) Thanks [@didoo](https://github.com/didoo)! 
```

## 1.4.1

**Month Day, Year**

> - Scope group layout styles to nested child components.
> 
> - Fix bug with `--token-pagination-child-spacing-vertical` value so that it adds "px" unit.
> 
> - [(#1204)](https://github.com/hashicorp/design-system/pull/1204) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!

Reviewer: I'm noticing that component and token changes in the same PR are turning out to be less of an edge case. Cleaning this up to include only the token changes and updating the formatting to add the PR link and contribution acknowledgment inline.

```suggestion
- Fixed a bug with `--token-pagination-child-spacing-vertical` value so that it adds "px" unit. [#1204](https://github.com/hashicorp/design-system/pull/1204) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!
```

## 1.4.0

**Month Day, Year**

> - `Pagination` add design tokens. [(#661)](https://github.com/hashicorp/design-system/pull/661) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! 

Reviewer: ```suggestion
- Added design tokens for `Pagination`. [#661](https://github.com/hashicorp/design-system/pull/661) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! 
```

## 1.3.1

**Month Day, Year**

- Update design system name to `Helios`. [(#1036)](https://github.com/hashicorp/design-system/pull/1036) Thanks [@Dhaulagiri](https://github.com/Dhaulagiri)!

## 1.3.0

**Month Day, Year**

> - Add design tokens for indeterminate `Checkbox`. [(#740)](https://github.com/hashicorp/design-system/pull/740) Thanks [@alex-ju](https://github.com/alex-ju)!

Reviewer: ```suggestion
- Added design tokens for indeterminate `Checkbox`. [#740](https://github.com/hashicorp/design-system/pull/740) Thanks [@alex-ju](https://github.com/alex-ju)!
```
